clipper_admin/clipper_admin/deployers/pytorch.py

```diff
@@ -207,17 +207,29 @@ def predict(model, inputs):
         # Check if Python 2 or Python 3 image
         if base_image == "default":
             if py_minor_version < (3, 0):
-                logger.info("Using Python 2 base image")
-                base_image = "{}/pytorch-container:{}".format(
-                    __registry__, __version__)
+                img_name = "pytorch-container"
+                if clipper_conn.cm.gpu:
```

Comment:
If `cm` is `KubernetesContainerManager`, isn't it throwing `AttributeError`? Because `KubernetesManager` doesn't have a flag `gpu`?  I think this parts should check
```
# Note the code is pseudo code
if clipper_conn.cm is DockerContainerManager and clipper_conn.gpu:
```

Or the altanative could be to add `gpu` flag to `KubernetesContainerManager` (if we will eventually offer this feature to Kubernetes), and throw an exception like `NotImplementedError` if the flag is on until we actually implement it.

Reply:
Will fix